Taskfile.yml

```diff
@@ -247,20 +247,26 @@ tasks:
       - >
         bin/ferretdb-testcover{{exeExt}} -test.coverprofile=cover.txt -- --test-records-dir=records
         --mode=diff-normal --listen-addr=:27017
+        --listen-tls=:47017 --listen-tls-cert-file=./build/certs/server-cert.pem 
+        --listen-tls-key-file=./build/certs/server-key.pem
 
   run-tigris:
     deps: [build-testcover]
     cmds:
       - >
         bin/ferretdb-testcover{{exeExt}} -test.coverprofile=cover.txt -- --test-records-dir=records
         --mode=diff-normal --listen-addr=:27017 --handler=tigris
+        --listen-tls=:47017 --listen-tls-cert-file=./build/certs/server-cert.pem 
```

Comment:
I remember there was a discussion what to run through TLS and what not. Is the final decision to run everything through TLS?

Reply:
The discussion was about tests